### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by szepeviktor**
Conclusions from #358

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -252,7 +252,7 @@
 - Added: Filter `SimpleHistoryFilterDropin/show_more_filters_on_load` that is used to control if the search options should be expanded by default when the history page is loaded. Default is false, to have a less cluttered GUI.
 - Added: Filter `SimpleHistoryFilterDropin/filter_default_user_ids` that is used to search/filter specific user ids by default (no need to search and select users). Should fix https://wordpress.org/support/topic/how-to-pass-array-of-user-ids-to-history-query.
 - Added: Filter `SimpleHistoryFilterDropin/filter_default_loglevel` that is used to search/filter for log levels by default.
-- Fixed: if trying to log an array or an object the logger now automagically runs `json_encode()` on the value to make it a string. Previously is just tried to run `$wpdb->insert() with the array and that gave errors. Should fix https://wordpress.org/support/topic/mysql_real_escape_string.
+- Fixed: if trying to log an array or an object the logger now automagically runs `json_encode()` on the value to make it a string. Previously is just tried to run `$wpdb->insert()` with the array and that gave errors. Should fix https://wordpress.org/support/topic/mysql_real_escape_string.
```
Comment (szepeviktor) [inline CHANGELOG.md:255]: A closing backtick.

File: code.md [hunk 2]
```diff
@@ -47,15 +43,19 @@ Config is in `phpstan.neon`.
...
 List errors and warnings:
 
-    $ phpcs /path/to/code/myfile.php # lint specific file
-    $ phpcs # be in plugin root and all files will be linted
-    $ npm run lint-php # or use npm script
+```bash
+phpcs /path/to/code/myfile.php # lint specific file
+phpcs # be in plugin root and all files will be linted
+npm run lint-php # or use npm script
+```
```
Comment (szepeviktor) [inline code.md:50]: Making shell commands copyable.

File: inc/class-simple-history.php [hunk 3]
```diff
@@ -2244,24 +2244,16 @@ public function settings_field_where_to_show() {
...
 
-		<input 
-		<?php
-		echo $show_on_dashboard
-			? "checked='checked'"
-			: '';
-		?>
-		 type="checkbox" value="1" name="simple_history_show_on_dashboard" id="simple_history_show_on_dashboard" class="simple_history_show_on_dashboard" />
+		<input
+			<?php checked($show_on_dashboard); ?>
```
Comment (szepeviktor) [inline inc/class-simple-history.php:2248]: Use core's `checked` function.

**Comment by bonny** [comment]
Thanks @szepeviktor

**Pull request merged**